alerts/UA00001.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dateRaised": 20200929,
+  "affectedFirmware": ["all"],
+  "hardwareLimited": [],
+  "description": "When using interrupt driven inputs on STM32 based boards, if an external hardware failure occurs that generates an extremely high interrupt rate (over 500,000 interrupts per second) the flight controller can become overloaded and cease to function. This affects the following types of inputs: RPM sensor with RPM_TYPE 1 or 2, Camera feedback with a CAM_FEEDBACK_PIN, Wheel encoders with WENC_TYPE=1, liquid fuel flow monitors with BATT_MONITOR=11 or 12, PWM rangefinder with RFND type 5, 22 or 30, RSSI with RSSI_TYPE=4.",
+  "mitigation": "Disable the parameter associated with the interrupt input",
+  "fixCommit": ["e471b23f5835be122dd461f7efe2a9ac05ee8faf", "965af98f4a2ed12ef76ff0c52d100f8cfad29e20", "c62b86e3c198212212ca84bb991a610a31ae8303"],
```

Comment:
will this git hash always be in the ArduPilot repo? or could it be on another repo/submodule.

Reply:
we are linear, so yes, it should always be in our repo, but maybe we should tag UserAlert fixes?

Reply:
so a chibios fix would contain the githash of the submodule pointer changing?

Reply:
Yes. For example this commit: https://github.com/ArduPilot/ardupilot/commit/ca167018232651f0eb88c15678deb77f7f964a77 will be represented as ``"fixCommit": ["ca167018232651f0eb88c15678deb77f7f964a77"]``